### ISSUE TITLE
Add new module: quote

### ISSUE THREAD

**Issue by swapagarwal** [the-vision/jarvis-telegram#10]
https://github.com/the-vision/jarvis-telegram/wiki

**Comment by VibavariG** [comment]
Sir, i would like to work on this issue

**Comment by RohitRathore1** [comment]
@VibavariG Thanks for your interest but at a time in the same repo you can work on one issue only.

**Comment by hritikksingh** [comment]
I would like to work on this, can you assign me this issue

**Comment by tanisha03** [comment]
@hritikpapa you can work now

**Comment by hritikksingh** [comment]
> @hritikpapa you can work now

thank you

**Comment by RohitRathore1** [comment]
@hritikpapa Are you working on it? If you have created PR, then please mention PR number in this issue or this issue number in your PR.

**Comment by hritikksingh** [comment]
I don't think so I can work on this now, I recently got my drive crashed and since can't buy a new one to start again.
You can assign this issue to someone else.

**Comment by Rukmini-Meda** [comment]
@swapagarwal @TeAmP0is0N I would like to work on this issue. Please assign this to me as part of GSSoC'20. Also, please explain the feature for me to start working on it.
